examples/scripts/spme_max_energy.py

```diff
@@ -17,6 +18,7 @@
 
 # Define parameter set and model
 parameter_set = pybop.ParameterSet.pybamm("Chen2020")
+set_formation_concentrations(parameter_set)
```

Comment:
I think let's keep this separate from the design optimisation classes for now. In future PRs, let's work towards a standalone "design parameter set".

Reply:
Sounds good, please open an issue when you get chance 👍

Reply:
In the meantime, I believe this could be an optional arg for `ParameterSet.pybamm()`. This would be a good stop gap until we expand with DesignParameters. Something like:

```python
parameter_set = pybop.ParameterSet.pybamm("Chen2020", set_concentrations="formation")
```
or
```python
parameter_set = pybop.ParameterSet.pybamm("Chen2020", set_formation_concentrations=True)
```

Reply:
Hi @BradyPlanden, I think this function may be useful after, as well as on, initialisation of a `pybamm.ParameterValues` object so I am hesistant to implement the above. The way `pybop.ParameterSet` is set up to return a `pybamm.ParameterValues` object means that it is difficult to add a function to this class at the moment. I think we should merge the current implementation, as the purpose here is to fix a bug, and open another issue to work on the `ParameterSet` class. What do you think?

Reply:
Could `set_formation_concentrations` be called within `ParameterSet.pybamm` before returning the pybamm parameter set if the optional argument is set? `set_initial_concentration` can then be a static method so that it can then be used independently as well.